synedrion/src/cggmp21.rs

```diff
@@ -36,6 +36,8 @@ pub use interactive_signing::{
 };
 pub use key_init::{KeyInit, KeyInitAssociatedData, KeyInitProtocol};
 pub use key_refresh::{KeyRefresh, KeyRefreshAssociatedData, KeyRefreshProtocol};
+#[cfg(test)]
+pub(crate) use params::TestParams32;
```

Comment:
Should have the same visibility as `TestParams` (which should perhaps be renamed to `TestParams64`?). `TestParams` usually would be something one would gate behind a `dev` feature, but it's just a single type, so I didn't bother creating a feature at the time.

Reply:
In my head we'd put `k256` behind a feature, then add a default feature and include `k256`. Then we'd pick one of the tiny-curves that we're happy with and always have that active in test mode. 

Adding `TestParams32` was more of a check that things work well with `TinyCurve32` (also so we don't accidentally build in a dependency on `TinyCurve64`).

I thought `TestParams` was used in tests outside `synedrion` too, in entropy-core?

Here's what I suggest we do:
- remove `TestParams32` from this PR
- leave `TestParams` (both the name and the pub export)
- add an issue to figure out the best "strategy" for the features we want to introduce in v0.3